src/groups/bmq/bmqa/bmqa_mocksession.cpp

```diff
@@ -234,12 +234,14 @@ Event MockSessionUtil::createSessionEvent(
         sessionEventType != bmqt::SessionEventType::e_QUEUE_CLOSE_RESULT &&
         sessionEventType != bmqt::SessionEventType::e_QUEUE_CONFIGURE_RESULT);
 
+    bslma::Allocator* alloc = bslma::Default::allocator(allocator);
+
     Event        event;
     EventImplSp& implPtr = reinterpret_cast<EventImplSp&>(
         static_cast<Event&>(event));
-    implPtr = EventImplSp(new (*allocator)
-                              bmqimp::Event(g_bufferFactory_p, allocator),
-                          allocator);
+    implPtr = EventImplSp(new (*alloc)
```

Comment:
Fixed some places where `new` was called with allocator pointer not wrapped with `bslma::Default::allocator`. These places were causing crashes when `allocator = 0` passed.